modules/webadmin.cpp

```diff
@@ -1567,6 +1569,13 @@ class CWebAdminMod : public CModule {
                 o12["Checked"] = "true";
             }
 
+            CTemplate& o13 = Tmpl.AddRow("OptionLoop");
```

Comment:
I think this option should be near 'change password' block, not hidden in random list of checkboxes below.